test/e2e-test/hack/addon-vela-test.sh

```diff
@@ -49,6 +49,22 @@ vela addon enable cert-manager
 vela addon enable flink-kubernetes-operator
 # set back to the DEFAULT_VELA_NS
 declare -x DEFAULT_VELA_NS=vela-system
+
+# test pyroscope addon
+# enable pyroscope
+kubectl create ns pyroscope-system
+declare -x DEFAULT_VELA_NS=pyroscope-system
+vela addon enable fluxcd
+vela addon enable pyroscope
```

Comment:
As discussed with @fourierr. These disable/enable fluxcd addon will waste too much time for running CI. The next pr will refactor the test script to reduce these operation.

Reply:
> As discussed with @fourierr. These disable/enable fluxcd addon will waste too much time for running CI. The next pr will refactor the test script to reduce these operation.

Yes, i will adjust the order of CI in next pr.